pkg/util/runtimeconfig/manager.go

```diff
@@ -165,10 +189,24 @@ func (om *Manager) loadConfig() error {
 	// expose hash of runtime config
 	om.configHash.Reset()
 	om.configHash.WithLabelValues(fmt.Sprintf("%x", hash[:])).Set(1)
-
 	return nil
 }
 
+func (om *Manager) loadConfigFromBucket(ctx context.Context) ([]byte, error) {
+	readCloser, err := om.bucketClient.Get(ctx, om.cfg.LoadPath)
```

Comment:
One question here, we are doing requests to the object storage every 15 seconds let's say.
Do you think we can have a way to avoid this request, if we know the content doesn't change?

For example, the object storage package defines the `Attributes` method which contians `LastModified` information. If we can cache the last time we load the config and check last modified time, then we can avoid loading the file everytime. We will call `Attributes` more but I think it is cheaper than downloading the file.

Reply:
It is a small optimization and we don't have to do it in this pr. The file is not large I assume so it should be ok.

Reply:
Good point, it seems more appropriate to cache and use the `LastModified` field for comparison since one of the key reasons for adding more options for storage backend is that we want to allow larger file sizes for the runtime config.

Reply:
We can do the optimization in another PR like you suggested. I'll resolve this comment for now.